recipes/hdfs3/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "0.1.1" %}
+
+package:
+  name: hdfs3
```

Comment:
> `python-hdfs3` might add more confusion to the namespace of HDFS-related libraries.

I don't think so. Most Linux distros do that. Renaming is confusing, but we are only suggesting a prefix :grin: